sources/packages/web/src/store/modules/auth/auth.ts

```diff
@@ -4,6 +4,7 @@ import { actions } from "./actions";
 import { mutations } from "./mutations";
 import { AuthState, RootState } from "../../states";
 
+
```

Comment:
This change does not seem part of this review. Is it possible to remove it?

Reply:
removed auth related files